### DISCUSSION TITLE
More information

### DISCUSSION THREAD

**Discussion by Volcannon** [boblord14/Dark-Souls-2-SotFS-CT-Bob-Edition#8] [Q&A]
Hello, I was wondering if there is more info on what some of these options actually do exactly? I know what some of the basic stuff does, like the stats, cheats, and the item param (for the most part) but some of them leave me completely confuzzled.

**Comment by boblord14** [comment]
Few answers here- 

- A good few things are modding tools- designed for people who understand the internal system and want to mess with it. Those parts are just sort of expected to only be used/messed with by those who know what they're doing. 
- If you open the scripts for certain things, they might have a note inside explaining what it does. This isn't true for everything, but is for the things I write(and the important bits iirc).
- I recommend you join the grand cathedral discord listed in the readme on the main page, as more direct questions can be asked and answered quicker.
